### DISCUSSION TITLE
Loss Curve, Validation Loss and Early Stopping

### DISCUSSION THREAD

**Discussion by chbardel** [PriorLabs/TabPFN#277] [Q&A]
The title says it all. How can I get training loss curves out of this? If I see it correctly this is not directly supported natively? Is there a reason for that and is it planned for the future?

Maybe someone can help me figure this out. Maybe there is a fairly simple code modification that I could do to support retaining loss curves. Preferably not only training loss, but also being able to evaluate on a validation set during training such that I can do early stopping etc. I have indications that on my data the model is heavily overfitting compared to other approaches.
